sql-to-dbsp-compiler/SQL-compiler/src/test/java/org/dbsp/sqlCompiler/compiler/sql/simple/RegressionTests.java

```diff
@@ -1497,10 +1518,10 @@ public void testSlt() {
     @Test
     public void issue3199() {
         this.statementsFailingInCompilation("""
-                CREATE TABLE t1(c0 VARCHAR);
+                CREATE TABLE t1(c0 VARBINARY);
                 CREATE TABLE t2(c0 DECIMAL(19, 9));
                 CREATE VIEW V AS (SELECT * FROM t1 NATURAL JOIN t2);""",
-                "Implicit conversion between VARCHAR and DECIMAL not supported");
+                "Column 'c0' matched using NATURAL keyword or USING clause has incompatible types");
```

Comment:
is it possible to output expected and actual types?

Reply:
The test only shows a substring of the error.
This is the full error message:
`Error in SQL statement: Column 'c0' matched using NATURAL keyword or USING clause has incompatible types: cannot compare 'VARBINARY' to 'DECIMAL(19, 9)'`

Reply:
nice